src/stravalib/tests/unit/test_lat_long.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from stravalib.model import LatLon, SummaryActivity
+
+
+@pytest.mark.parametrize(
+    "start_latlng, end_latlng, start_expected_return, end_expected_return",
+    [
+        (
+            [10.2, 1.02],
+            [10.2, 1.02],
+            LatLon(root=[10.2, 1.02]),
+            LatLon(root=[10.2, 1.02]),
+        ),
+        (None, None, None, None),
+        ([], [], None, None),
```

Comment:
This fails consistently. i have an activity that for some reason had an empty list - that is how i found this. so i thought a test would be warranted here and then we will want to fix this too. many thanks. i'm just not sure where the typing issue is coming from and how we can clean things up.